lib/codeowners/checker.rb

```diff
@@ -93,13 +112,36 @@ def defined_owner?(file)
       false
     end
 
+    def valid_owner?(owner)
+      return true unless @validate_owners
+
+      owners_list.include?(owner)
```

Comment:
What do you think about:
```suggestion
      !@validate_owners || owners_list.include?(owner)
```